setup.sh

```diff
@@ -3,7 +3,6 @@
 # List of commands to run:
 commands=(
   "lando rebuild -y"
-  "lando composer install"
```

Comment:
`lando rebuild` will trigger `composer install` already, so we can skip this step.